manifests/plugin/salt.pp

```diff
@@ -38,8 +41,24 @@
   String $api_password = $foreman_proxy::plugin::salt::params::api_password,
   Optional[Stdlib::Absolutepath] $saltfile = $foreman_proxy::plugin::salt::params::saltfile,
 ) inherits foreman_proxy::plugin::salt::params {
+  $foreman_ssl_cert = pick($foreman_proxy::foreman_ssl_cert, $foreman_proxy::ssl_cert)
+  $foreman_ssl_key = pick($foreman_proxy::foreman_ssl_key, $foreman_proxy::ssl_key)
+  $reactor_path = "/var/lib/foreman-proxy/salt/reactors"
```

Comment:
Is this file stored in the foreman_proxy_dir? See https://github.com/theforeman/puppet-foreman_proxy/blob/41b77855b9b9bf112ddcdd84f58aa65a201a858d/manifests/plugin/ansible.pp#L68

Reply:
No, it's a different directory which I didn't find an existing variable for.
`${foreman_proxy::config_dir}` points to `/etc/foreman-proxy`. Accordingly, the code you referenced handles `/etc/ansible/ansible.cfg` and `/etc/foreman-proxy/ansible.cfg`.

The `$reactor_path` is about `/var/lib/foreman-proxy/salt/reactors` which is the path for the Salt reactor files.